package.json

```diff
@@ -18,36 +18,37 @@
     "allowTokens": "./scripts/allowTokens",
     "allowedTokens": "./scripts/allowedTokens",
     "prettier:fix": "npx prettier --write --ignore-unknown .",
-    "prettier": "npx prettier --check ."
+    "prettier": "npx prettier --check .",
+    "switchPostInstall": "scripts/switchPostInstall"
```

Comment:
why do we need the `switchPostInstall` script?

Reply:
I recovered this change for mistake.
Removed.